_15thnight/core.py

```diff
@@ -95,10 +96,26 @@ def respond_to_alert(provider, needs_provided, alert):
     return response
 
 
-def send_out_resolution(need):
-    """
-    Sends out a resolution for a need to providers.
-    """
+def send_alert_resolution_notice(need):
```

Comment:
Move the code in this function into the bottom of `resolve_need`. No need for this to be in its own function.